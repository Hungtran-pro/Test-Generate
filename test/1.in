8
20
5
25
23
50
27
41
28
