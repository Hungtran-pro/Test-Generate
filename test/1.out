576873480
5683639
304394701
402392637
369711062
269420915
519664494
832229816
